frontend/src/pages/LoginRegister/RegisterPage.vue

```diff
@@ -31,6 +32,32 @@
   </div>
 </template>
 
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  data() {
+    return {
+      username: "",
+      password: "",
+      confirmPassword: "",
+    };
+  },
+  methods: {
+    register() {
+      if (this.password !== this.confirmPassword) {
+        // TODO: show some error here.
```

Comment:
For this we can probably use add a rule / validator using https://youzan.github.io/vant/#/en-US/form#validate-rules instead of manually handling it?